src/pages/singleLineMap/index.tsx

```diff
@@ -123,79 +121,70 @@ const SingleLineMapPage = () => {
         {t('singleline_map_page_title')}
         <InfoYoutubeModal
           label={t('open_video_about_this_page')}
-          title={t('youtube_modal_info_title')}
+          title={t('realtime_map_page_description')}
```

Comment:
I think it's not the description of the right page, and perhaps we should consider includith both title and paragraph in the modal